assets/components/LoadingPokemonCard.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { ActivityIndicator, View } from "react-native";
+import styled from "styled-components";
+
+const LoadingPokemonCard = () => {

```

Comment:
Maybe to make this component even more general it could be named only Loading or LoadingSpinner?
Then you could reuse it in any project you want without having to change anything to it.